tests/AbstractErrorHandlerTest.php

```diff
@@ -89,5 +118,5 @@ public function captureAtDataProvider()
         ];
     }
 
-    abstract protected function createErrorHandler(...$arguments);
+    abstract protected function createErrorHandler(...$arguments): AbstractErrorHandler;
```

Comment:
Instead of declaring the typehint here, we could omit it and declare the real error handler typehint in the method of the subclasses